homeassistant/components/hue/__init__.py

```diff
@@ -99,6 +101,20 @@ async def async_setup_entry(
         data.pop(CONF_ALLOW_HUE_GROUPS)
         hass.config_entries.async_update_entry(entry, data=data, options=options)
 
+    # Migrate default_scene_transition from config entry data to config entry options
```

Comment:
Why are you doing this? It's not used in the rest of the PR.

Reply:
I was thinking the default scene transition time should be configurable and this is where that updates correct?

Reply:
The above code is a migration after we moved some option around. It's not providing any option. Remove this.